src/components/JobsPanelParameters/JobsPanelParameters.js

```diff
@@ -61,7 +60,7 @@ const JobsPanelParameters = ({
     setAddNewParameter(false)
     setNewParameter({
       name: '',
-      type: '',
+      type: 'string',
```

Comment:
You should either use id here and get label wherever you need. Or use label here. If you use label here you can't hardcode it.